api/src/app.ts

```diff
@@ -1,6 +1,30 @@
-import express, { Express, Request, Response } from 'express';
 import dotenv from 'dotenv';
-import PostRepository from './repositories/posts/PostRepository';
+import express, { Express } from 'express';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+import { version } from '../package.json';
+import posts from './routes/Post';
+
+const swaggerOptions: swaggerJSDoc.Options = {
+    definition: {
+        openapi: '3.0.0',
+        info: {
+            title: 'Sweep API',
+            version,
+            contact: {
+                email: 'giesscollin@gmail.com',
+                name: 'Collin Giess',
+            },
```

Comment:
I think this should be extracted to another file in case the contact changes, enabling easy configurability.

Reply:
Working on this now